components/layouts/content-list/content-list--years.hbs

```diff
@@ -0,0 +1,79 @@
+      <p>Because we don’t manufacture any goods, our carbon footprint is largely
+        driven by office utilities, employee commuting, business travel, and
+        technology systems.</p>
+
+      <p>Taking these factors into account and using the
+        <a
+          href="https://www.epa.gov/climateleadership/scope-1-and-scope-2-inventory-guidance"
+        >EPA guidelines for calculating</a>
+        carbon generation, we’ve calculated our company’s global carbon
+        footprint (including internal staff and talent) for 2019, 2020, and 2021
+        (projected):</p>
+    </div>
+
+    <dl class="content-years">
+      <dt class="content-years__title-group">
+        <h2 class="content-years__year">2019</h2>
+        <div class="content-years__stat-description">
+          <h3 class="content-years__stat">Benchmark</h3>
+        </div>
+        <hr class="content-years__connector" />
+      </dt>
+      <dd class="content-block">
+        <div class="themed-text">We generated almost
+          <span class="content-years__text-highlighted">9,000 tons</span>
+          of carbon. A substantial portion of this number is due to our staff
+          and talent commuting to offices.</div>
+      </dd><!-- .content-block -->
+    </dl><!-- .content-years -->
+
+    <dl class="content-years">
+      <dt class="content-years__title-group">
+        <h2 class="content-years__year">2020</h2>
+        <div class="content-years__stat-description">
+          <h3 class="content-years__stat">88% Carbon Reduction</h3>
+          <span class="content-years__attribute">Since 2019</span>
+        </div>
+        <hr class="content-years__connector" />
+      </dt>
+      <dd class="content-block">
+        <div class="themed-text">A pivotal year, with COVID forcing us to move
+          all our staff and talent to work remotely, we generated just over
+          <span class="content-years__text-highlighted">1,100 tons</span>
+          of carbon.</div>
+      </dd><!-- .content-block -->
+    </dl><!-- .content-years -->
+
+    <dl class="content-years">
+      <dt class="content-years__title-group">
+        <h2 class="content-years__year">2021</h2>
+        <div class="content-years__stat-description">
+          <h3 class="content-years__stat">96% Carbon Reduction</h3>
+          <span class="content-years__attribute">Since 2019</span>
+        </div>
+        <hr class="content-years__connector" />
+      </dt>
+      <dd class="content-block">
+        <div class="themed-text">This year, we estimate our carbon footprint
+          will be approximately
+          <span class="content-years__text-highlighted">360 tons</span>
+          as we let go of physical offices and became a
+          <a
+            href="https://aquent.com/news/aquent-goes-100-percent-virtual"
+          >virtual-first workplace</a>.</div>
+      </dd><!-- .content-block -->
+    </dl><!-- .content-years -->
+
+    <div class="content-years__postface themed-text">
```

Comment:
As mentioned above, let's ditch the wrapper, just assume we're displaying a plain old `p` from a basic content item in the CMS.